src/utils/v4_position_planner.rs

```diff
@@ -0,0 +1,131 @@
+use super::v4_planner::{Actions, V4Planner};
+use crate::{
+    abi::{
+        BurnPositionParams, DecreaseLiquidityParams, IncreaseLiquidityParams, MintPositionParams,
+        SettlePairParams, SweepParams, TakePairParams,
+    },
+    entities::Pool,
+};
+use alloy_primitives::{Address, Bytes, U256};
+use uniswap_sdk_core::prelude::BaseCurrency;
+use uniswap_v3_sdk::prelude::TickIndex;
+
+#[derive(Clone, Debug, Default, PartialEq)]
```

Comment:
```suggestion
#[derive(Clone, Debug, Default, PartialEq, Deref)]
```
Import `Deref` from `derive_more`.